src/server/command_registry.cc

```diff
@@ -52,12 +56,23 @@ bool CommandId::IsTransactional() const {
 }
 
 void CommandId::Invoke(CmdArgList args, ConnectionContext* cntx) const {
-  uint64_t before = absl::GetCurrentTimeNanos();
-  handler_(std::move(args), cntx);
-  uint64_t after = absl::GetCurrentTimeNanos();
+  int32_t log_slower_than = absl::GetFlag(FLAGS_slowlog_log_slower_than);
```

Comment:
this function is on critical path to every request so I will have to request a very efficient code here :) 

1. introduce a variable `ServerState* ss = ServerState::tlocal();` and use it everywhere.
2. We can not use `GetFlag` here. It's not efficient enough for a 10M qps server.  Therefore, we will need to move this configuration variable `log_slower_than` to ServerState. Also rename it to `log_slower_than_usec`.
3. `cntx->owner()` is null, do not bother recording the slow log.
4. It will be more elegant if SlowLog::Add will accept `std::string_view` and then you won't need to do all these conversions.

Reply:
I'm not sure if `cntx->owner`/ cntx->conn has any effect right now since it's set in the tests as well.

Reply:
you are right, missed that